pkg/scyllaclient/client_scylla.go

```diff
@@ -823,3 +823,57 @@ func (c *Client) TableDiskSizeReport(ctx context.Context, hostKeyspaceTables Hos
 
 	return report, err
 }
+
+// LoadSSTables that are already downloaded to host's table upload directory.
+func (c *Client) LoadSSTables(ctx context.Context, host, keyspace, table string, loadAndStream, primaryReplicaOnly bool) error {
+	const restoreTimeout = 15 * time.Minute // TODO: should we even set that? To what value?
```

Comment:
Good point.
What you do here is just setting timeout on API call. It protect the request from hanging forever.
According to https://github.com/scylladb/scylla-manager/blob/58f469cdb00b24d0c30c3c509f9a1cc33dc54042/pkg/service/backup/restore_worker_data.go#L637
this API call is async (or it behaves this way).

I would say there is completely no need of setting this value to 15 minutes.
I would even say value of this timeout is not so important. 

I propose to lower the value to 1m, 15 minutes is not harmful still tough.

Reply:
Calling load_and_stream when no load_and_stream is running is sync - response will not be sent until load_and_stream finishes.
Calling load_and_stream when other load_and_stream is running is async - we immediately get error response that says, that load_and_stream is already running.
That's why I am hesitating on this timeout value, but as you said, for now it should be left for QA team to decide what to do with the timeout.

Reply:
OK, as it's confirmed that load&stream scylla API call is sync.